tests/unit/amo/components/TestAddon.js

```diff
@@ -1238,6 +1239,46 @@ describe(__filename, () => {
 
     expect(root.find('title')).toHaveLength(0);
   });
+
+  describe('Contribute', () => {
+    it('does not render a Contribute card when there is no add-on', () => {
+      const root = shallowRender({ addon: null, params: { slug: 'some-slug' } });
+
+      expect(root.find(Contribute)).toHaveLength(0);
+    });
+
+    it('does not render a Contribute card when there is no contributions URL', () => {
```

Comment:
Again, these seem like tests that should be moved to the `Contribute` component. I think if we had it deal with the `addon` object as a whole and just render `null` when things are missing that'd be good and this component would need fewer tests.